modules/shortcodes/gist.php

```diff
@@ -128,14 +201,13 @@ function github_gist_shortcode( $atts, $content = '' ) {
 
 /**
  * Use script tag to load shortcode in editor.
+ * Can't use wp_enqueue_script here.
  *
  * @since 3.9.0
  *
  * @param string $id The ID of the gist.
- *
- * @return string
  */
 function github_gist_simple_embed( $id ) {
 	$id = str_replace( 'json', 'js', $id );
-	return '<script type="text/javascript" src="https://gist.github.com/' . $id . '"></script>';
+	return '<script type="text/javascript" src="https://gist.github.com/' . $id . '"></script>'; // phpcs:ignore WordPress.WP.EnqueuedResources.NonEnqueuedScript
```

Comment:
`type` in `<script>` tags is HTML4. 🤔 Maybe drop it while at it?

Reply:
👍 Done in d855731b31dc20cb844b8d55d443814f21485eee